website/content/docs/security/acl/index.mdx

```diff
@@ -67,11 +67,23 @@ As a result, roles can provide a more convenient authentication infrastructure t
 
 Refer to the [Roles](/consul/docs/security/acl/acl-roles) topic for additional information.
 
+## Templated Policies
+
+Templated policies are a configuration block that you can add to token and role configurations. You can use them during the authorization process to automatically generate a policy for specific use-cases. Consul links the generated policy to the role or token, which provides the required access to a particular resource.
+
+Templated policies enable you to quickly construct policies for common use cases, rather than creating multiple identical policies or digging through the permission system to understand permissions.
```

Comment:
```suggestion
Templated policies are configuration blocks that you can add to token and role configurations. Consul links the generated policy to the role or token, which provides the required access to a particular resource. 

You can use them during the authorization process to automatically generate a policy for common use-cases, rather than creating multiple identical policies or digging through the permission system to understand permissions. 
```
I realized that we were saying conflicting things in each paragraph. I think this is a little clear.